src/Language/C/Data/Ident.hs

```diff
@@ -102,7 +103,7 @@ bits28 = 2^(28::Int)
 --
 -- * for reasons of simplicity the complete lexeme is hashed.
 mkIdent            :: Position -> String -> Name -> Ident
-mkIdent pos s name  = Ident s (quad s) (mkNodeInfo' pos (pos,length s) name)
+mkIdent pos s name  = s `deepseq` Ident s (quad s) (mkNodeInfo' pos (pos,length s) name)
```

Comment:
We could probably avoid a deepseq dependency by just handwriting the seq instance for lists.